### PR TITLE
[Lua] Add warning/reminder to function header

### PR DESCRIPTION
**Pull request by Xaver-DaRed**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Adds a reminder to update function in core when/if updating function parameters in lua.

## Steps to test these changes

None

### CODE REVIEW

**Review by WinterSolstice8 — DISMISSED** [review]

**Comment by WinterSolstice8** [comment]
has conflicts apparently

**Comment by Xaver-DaRed** [comment]
Rebased

File: scripts/globals/combat/physical_utilities.lua [hunk 1]
```diff
@@ -381,12 +384,12 @@ xi.combat.physical.calculateMeleePDIF = function(actor, target, weaponType, wsAt
...
-    local pDifUpperCap = 0
-    local pDifLowerCap = 0
-    local damageLimitPlus = actor:getMod(xi.mod.DAMAGE_LIMIT) / 100
-    local damageLimitPercent = (100 + actor:getMod(xi.mod.DAMAGE_LIMITP)) / 100
-    local pDifFinalCap = (xi.combat.physical.pDifWeaponCapTable[weaponType][1] + damageLimitPlus) * damageLimitPercent + (isCritical and 1.0 or 0) -- Added damage limit bonuses
+    local wRatio             = cRatio + (isCritical and 1 or 0)
+    local pDifUpperCap       = 0
+    local pDifLowerCap       = 0
+    local damageLimitPlus    = actor:getMod(xi.mod.DAMAGE_LIMIT) / 100
+    local damageLimitPercent = 1 + actor:getMod(xi.mod.DAMAGE_LIMITP) / 100
```
Comment (WinterSolstice8) [inline scripts/globals/combat/physical_utilities.lua:391]: This +1 here would mean all pdif caps are increased by 1.0

Comment (Xaver-DaRed) [reply scripts/globals/combat/physical_utilities.lua:391]: No logic was changed

1 + X/100 = (100 + x) / 100

This is suposed to be a rate multiplier, meaning 1 == no change

Comment (WinterSolstice8) [reply scripts/globals/combat/physical_utilities.lua:391]: hmm needs more thought, hold on

Comment (WinterSolstice8) [reply scripts/globals/combat/physical_utilities.lua:391]: Nevermind, I thought that was the additive damage limit not the multiplicative. Long work day, sorry.

**Review by WinterSolstice8 — APPROVED** [review]

**Pull request merged**